lstmcpipe/stages/mc_r0_to_dl1.py

```diff
@@ -129,6 +129,7 @@ def r0_to_dl1(
     workflow_kind="lstchain",
     keep_rta_file=False,
     n_jobs_parallel=20,
+    debug_mode=False,
```

Comment:
I see no way to set this, how do you want to use it?

Reply:
hhmm right... 🤔

Reply:
Indeed, there are functionalities that were implemented long time ago, but now I'm not sure they make much sense.... (before you could configured HiPeRTA from the config file...). Same for `keep_files` indeed.

Reply:
I would propose.
Nobody uses hiperta functionality but me. 
Let's not add extra args to the config file that will never be used. 
Either I implement some optional args when `hiperta` workflow is set (and open an issue otherwise I will forgot), either (what would be most probable) I change the default value of the vars on the script by hand and I re-install 🙈 🙈 🙈 🙈  - not that I do this always 😂